.github/workflows/update-libs.yaml

```diff
@@ -1,53 +0,0 @@
-name: Auto-update Charm Libraries
```

Comment:
There is no way to auto-update charm libraries, is that right ?

Reply:
OK, the next PR adding it.

Reply:
This will be done from the scheduled workflow in the main branch.
I forgot to add it tho, so thanks!